common/errors.js

```diff
@@ -217,6 +219,52 @@
         ForbiddenAssetAccess.prototype = Object.create(Error.prototype);
         ForbiddenAssetAccess.prototype.constructor = ForbiddenAssetAccess;
 
+        /**
+         * Error class that is used when a user logs , but the page's previous session was another user
+         *
+         * @return {object}        Error Object
+         */
+        function DifferentUserConflictError(sessionInfo, prevSessionInfo) {
```

Comment:
The options for the error based on the drawing that Karl did were "reload" and "continue". Based on my assumption the "continue" button was meant to do the session check again and if the issue resolved, the error should be dismissed automatically to allow users to get back to their old state.

We could reuse the "ok" button that currently is directing users to the home page for this purpose.

(This comment doesn't necessarily belong to this part of the code and I added here because it's related to this new error class)